### PR TITLE
Added fleet model.  Closes #3

### LINKED ISSUE #3: Create Fleet data Model
Create a fleet for the airline. Budget is $10M. 
Click [here](https://wacky.jlparry.com/info/airplanes) to see available planes. Price is in $ thousands.

Need a unique identifier for each airplane, and the kind of plane it is. Any other information is up to you.
Unique identifiers should be alphanumeric, starting with the same first letter as our team name.

Mock the data in a model class.

### PR DESCRIPTION
**Pull request by spennydl**

### CODE REVIEW

File: application/models/Fleet.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Dummy model class that represent our airline's fleet
+ */
+class Fleet extends CI_Model
+{
+
+    // The mock plane data
+	var $data = array(
```
Comment (achlin) [inline application/models/Fleet.php]: There's some inconsistent tabs spacing in this whole file. Please double check. Yes I know this is very picky.

**Review by achlin — CHANGES_REQUESTED** [review]

File: application/models/Fleet.php (hunk 1 above)
Comment (spennydl) [reply application/models/Fleet.php]: i admire your attention to detail.
shoudl we go with 4 space indents everywhere?

Comment (achlin) [reply application/models/Fleet.php]: Ok let's go with 4 space indents.

**Review by achlin — APPROVED** [review]
Looks good.

**Pull request merged**